### PR TITLE
Add popup menu

### PR DESCRIPTION
**Pull request by Kimutai01**
-Add js popup menu for desktop
-Add js popup menu for mobile
-Add see project functionality

### CODE REVIEW

File: script/script.js [hunk 1]
```diff
@@ -34,3 +34,63 @@ lin.addEventListener('click', () => {
...
+<li class="desktop">${project.language4}</li>
+<li class="desktop">${project.language5}</li>
+<li class="desktop">${project.language6}</li>
+<li class="desktop">${project.language7}</li>
+<li class="desktop">${project.language8}</li>
+</ul>
+<p class="description">${project.description}</p>
+<div class="buttons non">
+<button>See Live<span><img src="assets/images/live.svg" alt="github"></span></button>
+<button>See Source<span><img src="assets/images/source.png" alt="github"></span></button>
```
Comment (brytebee) [inline script/script.js]: _**On mobile **_
- [ ] **Required:** You have done a good job so far, however, I observed that you have not followed the Figma template in position some elements. Take a look below:

Your Design | Template
:---: | :---:
image | image

- Kindly remove the white space at the top and add padding to the image.

Your Design | Template
:---: | :---:
image | image

- Reposition the button and add padding to them appropriately 

**_On desktop_**

Your Design | Template
:---: | :---:
image | image

- Kindly reposition the labeled elements on the popup.

File: script/script.js [hunk 2]
```diff
@@ -34,3 +34,63 @@ lin.addEventListener('click', () => {
...
+  image: 'assets/images/Snapshoot.svg',
+  description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  language1: 'Github',
+  language2: 'Codepen',
+  language3: 'JavaScript',
+  language4: 'Bootstrap',
+  language5: 'css',
+  language6: 'Terminal',
+  language7: 'Codekit',
+  language8: 'Ruby on Rails',
```
Comment (brytebee) [inline script/script.js]: - [ ] **Required:** Kindly place all languages into an array and loop through them.
image

_**Hint:**_ `projects.forEach((project) => project.forEach((language) => console.log(language.language)));`

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Kimutai01👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- Linters correctly setup✔️
- The modal window pops up and it's responsive✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script/script.js (hunk 2 above)
Comment (Kimutai01) [reply script/script.js]: i have used an array

Comment (mateo951) [reply script/script.js]: Addressed ✔

File: script/script.js (hunk 1 above)
Comment (mateo951) [reply script/script.js]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi  @Kimutai01, 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🔥
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**